pkg/types/defaults/machinepools.go

```diff
@@ -26,15 +26,20 @@ func SetMachinePoolDefaults(p *types.MachinePool, platform string) {
 	}
 }
 
-// CreateEdgeMachinePoolDefaults create the edge compute pool when it is not already defined.
-func CreateEdgeMachinePoolDefaults(pools []types.MachinePool, platform string, replicas int64) *types.MachinePool {
+// HasEdgePoolConfig checks if the Edge compute pool has been defined on install-config.
+func HasEdgePoolConfig(pools []types.MachinePool) bool {
```

Comment:
nit: does this need to be exported or created if it is only used once in this package?